libs/platform/src/lib/components/form/checkbox/checkbox.component.ts

```diff
@@ -3,21 +3,30 @@ import { Input, NgZone, Optional, Output, Self, ViewChild, ViewEncapsulation } f
 import { NgControl, NgForm } from '@angular/forms';
 import { CheckboxComponent as CoreCheckboxComponent } from '@fundamental-ngx/core';
 import { BaseInput } from '../base.input';
-import { FormFieldControl } from '../form-control';
+import { FormFieldControl, Status } from '../form-control';
 
 /** Change event object emitted by Platform Checkbox. */
 export class PlatformCheckboxChange {
     /** The source Checkbox of the event. */
     source: CheckboxComponent;
-    /** The new `checked` value of the checkbox. */
-    checked: boolean;
+    /**
+     * The new `checked` value of the checkbox.
+     * possible value: true/false and array of checkbox values.
+     */
+    checked: any;
```

Comment:
I think checked should stay as boolean and user will pass something like
[checked] = "whateverValue === condition"

Reply:
It's checkbox value, so it can be any other type as well